### PR TITLE
[ci] Temporary fix for CompCert

### PR DESCRIPTION
**Pull request by ejgallego**
Temporal fix while we wait for the resolution of https://github.com/AbsInt/CompCert/issues/234

### CODE REVIEW

**Review by ppedrot — COMMENTED** [review]
It's a pretty urgent PR.

**Comment by ejgallego** [comment]
> It's a pretty urgent PR.

I'm sure Gaëtan will be responsive.

**Comment by ejgallego** [comment]
Ahh CI seems stuck.

File: dev/ci/ci-compcert.sh [hunk 1]
```diff
@@ -5,6 +5,10 @@ ci_dir="$(dirname "$0")"
 
 CompCert_CI_DIR="${CI_BUILD_DIR}/CompCert"
 
+# Temporal workaround, to be removed when upstream decides what to do
```
Comment (SkySkimmer) [inline dev/ci/ci-compcert.sh]: Doesn't "temporal" mean some church-state concept? And something about brain parts.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
It works, I'm not going to wait for the other jobs.

File: dev/ci/ci-compcert.sh (hunk 1 above)
Comment (ejgallego) [reply dev/ci/ci-compcert.sh]: I will never learn.

**Comment by Zimmi48** [comment]
Yeah so indeed, it is temporary that was meant here :laughing:

**Comment by Zimmi48** [comment]
This "temporal" fix didn't seem to work when trying to backport: https://gitlab.com/Zimmi48/coq/-/jobs/72919337

It did in Travis though: https://travis-ci.org/Zimmi48/coq/jobs/388858581.

Anyways, the problem has been solved upstream in the meantime.

**Pull request merged**